docs/next.config.js

```diff
@@ -5,4 +5,13 @@ module.exports = {
   experimental: {
     externalDir: true,
   },
+  async redirects() {
+    return [
+      {
+        source: '/docs',
+        destination: '/docs/introduction',
```

Comment:
Should we introduce a `routes` object and type? Something like:

```ts
// routes.ts
export const routes = {
  home: '/',
  docs: '/docs',
  'docs-introduction': '/docs/introduction',
} as const;

export type Routes = typeof routes;
```

Reply:
It isn't super clear to me what benefit this will bring? Perhaps if we start referencing routes elsewhere? But until that time I'd be inclined to keep it simple.